static/js/subs.js

```diff
@@ -727,6 +727,43 @@ $(function () {
         update_announce_stream_state();
     });
 
+    // Search People
+    $(document).on('input', '.add-user-list-filter', function (e) {
+        var users = people.get_rest_of_realm();
+        var i;
```

Comment:
Declaring inside loop was raising lint error. `Move 'var' declarations to the top of the function.`
Though, I am updating this loop using `_.each `. This will be more readable and more consistent with zulip codebase/style.